cLvgl/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+services:
+    __container__-debug:
+        build:
+            context: .
+            dockerfile: Dockerfile.debug
+        image: ${LOCAL_REGISTRY}:5002/__container__-debug:${TAG}
+        ports:
+            - ${DEBUG_SSH_PORT}:${DEBUG_SSH_PORT}
+        devices:
+            - /dev/fb0:/dev/fb0
```

Comment:
Here I think that would be better use the cgroups rules:

```
# Add device access rights through cgroup...
    device_cgroup_rules:
      # ... for tty
      - "c 4:* rmw"
      # ... for /dev/input devices
      - "c 13:* rmw"
      # ... for /dev/dri devices
      - "c 226:* rmw"
      - "c 199:* rmw"
      # ... for /dev/fb0
      - "c 29:* rmw"
```

and share the `/dev`:

```
volumes:
      - type: bind
        source: /dev
        target: /dev
```